.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+# Have you read capa's CONTRIBUTING guide?
+It contains helpful information about how to contribute to capa. Check https://github.com/fireeye/capa/blob/master/.github/CONTRIBUTING.md
+
+based on https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/
+-->
+
+### Description
+
+closes # (issue)
+
+<!-- Description of the PR -->
+
+### Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
Checklists in PR sound like a good idea, but my experience is that in practice they are just removed because they are annoying to fill. I am up to give it a try, but I am not sure if we need a checklist for the type of change if soon it is expected that the author updates the changelog (which has this information).

Reply:
I like this part to get a quick overview of what the PR provides. I've removed the actual Checklist entry.